cachi2/core/models/output.py

```diff
@@ -14,7 +14,7 @@ class Property(pydantic.BaseModel):
     """A property inside an SBOM component."""
 
     name: str
-    value: str
+    value: Union[str, bool]
```

Comment:
property values should be strings, so we want `"true"` rather than `True`

https://cyclonedx.org/docs/1.4/json/#components_items_properties_items_value

Reply:
Doh! I saw boolean somewhere and apparently never looked back :sweat_smile:

Should be fixed